plugins/modules/k8s_service.py

```diff
@@ -213,28 +214,35 @@ def argspec():
 
 def perform_action(svc, resource, definition, params):
```

Comment:
It looks to me like we can just replace this function with a call to `module_utils.k8s.runner.perform_action()`.